libmscore/measure.cpp

```diff
@@ -4011,7 +4011,7 @@ void Measure::removeSystemTrailer()
             changed = true;
             }
       setTrailer(false);
-      if (changed)
+      if (system() && changed)
```

Comment:
What's the purpose of this extra check?  Is it just an optimization because we know we're going to recompute the width later anyhow?  My recollection is system() will be null while we are still testing whether the measure fits, I'd have thought we needed to get the width correct here.

Reply:
Well, I could have put the check in computeMinWidth() instead. The purpose is to make sure that the null system is not dereferenced in computeMinWidth().

Reply:
Usually when this function is called, `this` already has a valid system. That is not the case if this function is called from createMMRest(). I considered copying most of the code from removeSystemTrailer() and pasting it into createMMRest(), but I opted for this approach.

Reply:
Yes, you're right, my mistake in thinking this was normally being called before the measure is added to the system.

BTW, I know I've seen this behavior - leftover trailer at end of score - before.  I can't say for sure if this was the exact scenario or not.